src/IR.cpp

```diff
@@ -583,6 +583,8 @@ Call::ConstString Call::if_then_else = "if_then_else";
 Call::ConstString Call::glsl_texture_load = "glsl_texture_load";
 Call::ConstString Call::glsl_texture_store = "glsl_texture_store";
 Call::ConstString Call::glsl_varying = "glsl_varying";
+Call::ConstString Call::coordinates_load = "coordinates_load";
```

Comment:
Adding a preposition would help clarify what the intrinsic does- Maybe "load_at_coordinate"?  Otherwise it sounds like we are loading from a thing called "coordinates", like the glsl_texture_load loads from a texture.